en/docs/cli/index.md

```diff
@@ -15,3 +15,7 @@ While all of the available commands are provided here, in alphabetical order, so
 - [`yarn install`]({{url_base}}/docs/cli/install): installs all the dependencies defined in a `package.json` file.
 - [`yarn publish`]({{url_base}}/docs/cli/publish): publishes a package to a package manager.
 - [`yarn remove`]({{url_base}}/docs/cli/remove): removes an unused package from your current package.
+
+## Default Command
```

Comment:
You need to append `<a class="toc" id="toc-default-command" href="#toc-default-command"></a>`